build.py

```diff
@@ -134,6 +147,30 @@ def main(*argv):
             "--cache-dir", cache_dir
         ])
 
+        if hash_names:
+            for fn in os.listdir(out_tmp_dir):
+                fn = os.path.join(out_tmp_dir, fn)
+                if os.path.isfile(fn):
+                    print(
+                        "Computing hashes for \"%s\"." % os.path.basename(fn)
+                    )
+
+                    fn_hashes = {}
+                    for hn in hash_names:
+                        h = subprocess.check_output(
+                            ["openssl", hn, fn],
+                            universal_newlines=True
+                        )
+                        h = h.split()[-1]
+                        fn_hashes[hn] = h
+
+                    print("Writing out hashes.")
+                    for hn in hash_names:
+                        h = fn_hashes[hn]
+                        with open(fn + os.extsep + hn, "w") as fh:
+                            fh.write(h)
+                            fh.write("\n")
```

Comment:
Dump all hashes into files like `miniforge.sh.sha256` for the `sha256` checksum of `miniforge.sh`.

Reply:
Also ensure there is a terminal newline for those that want to `cat` the file.